content/en/user-guide/aws/athena/index.md

```diff
@@ -177,7 +177,28 @@ The `SELECT` statement above currently requires us to prefix the database/table
 
 ## Iceberg Tables
 
-The LocalStack Athena implementation also supports [Iceberg tables](https://docs.aws.amazon.com/athena/latest/ug/querying-iceberg-creating-tables.html) - more details and samples will be provided here soon.
+The LocalStack Athena implementation also supports [Iceberg tables](https://docs.aws.amazon.com/athena/latest/ug/querying-iceberg-creating-tables.html). You can define an Iceberg table in Athena using the `CREATE TABLE` statement, as shown in the example below:
```

Comment:
Please add each sentence to it's own line (as stated in the best practice in the README of the repo). It does not make a difference in how it's displayed, but it immensely simplifies the reviews:
```suggestion
The LocalStack Athena implementation also supports [Iceberg tables](https://docs.aws.amazon.com/athena/latest/ug/querying-iceberg-creating-tables.html).
You can define an Iceberg table in Athena using the `CREATE TABLE` statement, as shown in the example below:
```